corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -15,36 +15,28 @@ hqDefine('analytix/js/hubspot', [
 ) {
     'use strict';
     var _get = initialAnalytics.getFn('hubspot'),
-        _global = initialAnalytics.getFn('global'),
-        logger = logging.getLoggerForApi('Hubspot'),
-        _data = {};
+        _logger,
+        _ready;
 
     var _hsq = window._hsq = window._hsq || [];
 
-    var __init__ = function () {
-        _data.apiId = _get('apiId');
-        if (_data.apiId) {
-            _data.scriptSrc = '//js.hs-analytics.net/analytics/' + utils.getDateHash() + '/' + _data.apiId + '.js';
-            utils.insertScript(_data.scriptSrc, logger.debug.log, {
-                id: 'hs-analytics',
```

Comment:
This also gets lost in the refactor, which I'm pretty sure is okay - the id is here because it's supplied in HubSpot's example code, but we never use it and I'm skeptical that the script references its own id. Will find out in testing.